lib/Sema/SemaResolveType.cpp

```diff
@@ -632,10 +661,14 @@ void TypeResolver::visit(CallExprType &CE) {
       case Identifier::SpecialIdentifier::abi_encodeWithSignature: {
         auto &Signature = CE.getRawArguments().at(0);
         if (auto *IC = dynamic_cast<ImplicitCastExpr *>(Signature.get())) {
-          Actions.resolveImplicitCast(*IC, std::make_shared<BytesType>(),
+          Actions.resolveImplicitCast(*IC, std::make_shared<StringType>(),
```

Comment:
Is the signature type `BytesType`? Why does this change from `BytesType` to `StringType`?

Reply:
StringType cannot ImplicitCast to BytesType in solidity, so I use ExplicitCast here.